teams/vsp/teams/triage/call-center/ROE-test.md

```diff
@@ -0,0 +1,75 @@
+# Preface
+This document will help all team members working on VA.gov understand how and when it is appropriate to engage the Call Center team for support and provide relevant information to assist in routing reported issues to other appropriate teams.  Because of the limited number of resources on Call Center, they cannot be a "catch-all" for every issue reported on the platform and every application.  Please read the following document carefully in order to determine the appropriate path for your incident.
+
+## IRIS Reporting
+
+VA.gov technical issues are routed to the Call Center IRIS inbox from the Contact Us page on VA.gov.  The goal is to either assign, reassign, or respond within 24 business hours.
+* Call Center team will review each issue to determine:
+    * If it needs to be escalated to the appropriate team
+    * If it should be reassigned to the Call Center that routed it to us
+    * If it can be responded to from the response template that has been created
+* The Call Center team will only report "trending" issues to the appropriate teams 
+    * The Call Center defines a trend as five or more of the same issue.
+    
+    
+## SalesForce Reporting
+
+VA.gov issues are captured by WHH (White House Hotline) agents in SalesForce.  The goal is to either assign, reassign, or respond within 24 business hours.
+* Call Center team will review each issue to determine:
```

Comment:
Call center team should only review any issue tagged as VA.gov, otherwise it will be too many issues to review.